tests/Integration/Behaviour/Features/Context/Domain/AliasFeatureContext.php

```diff
@@ -183,6 +185,39 @@ public function bulkDeleteAlias(string $aliasReferences): void
         }
     }
 
+    /**
+     * @When I search for alias search term matching :search I should get the following results:
+     *
+     * @param string $search
+     * @param TableNode $tableNode
+     */
+    public function assertSearchAliases(string $search, TableNode $tableNode): void
+    {
+        /** @var AliasForAssociation[] $foundAliasesForAssociation */
+        $foundAliasesForAssociation = $this->getQueryBus()->handle(new SearchAliasesForAssociation($search));
+        $expectedSearchTermsRows = $tableNode->getColumnsHash();
+
+        foreach ($expectedSearchTermsRows as $expectedSearchTermRow) {
+            $expectedSearchTerms = PrimitiveUtils::castStringArrayIntoArray($expectedSearchTermRow['searchTerm']);
+            Assert::assertEquals(count($expectedSearchTerms), count($foundAliasesForAssociation));
```

Comment:
```suggestion
            Assert::assertCount(count($expectedSearchTerms), $foundAliasesForAssociation, 'Expected and found search terms count doesn't match');
```
its nice to have those additional messages to help understand what exactly fails